src/jorphan/org/apache/jorphan/math/StatCalculator.java

```diff
@@ -162,7 +162,7 @@ public T getPercentPoint(double percent) {
         }
 
         // use Math.round () instead of simple (long) to provide correct value rounding
-        long target = Math.round (count * percent);
+        long target = Math.round(Math.ceil(count * percent));
```

Comment:
No need to call `Math.round()` after `Math.ceil()`. A simple cast to `long` would be enough. But it will probably not hurt, either.

But you should adapt the comment above the line to the new algorithm.